src/main/java/fr/insee/genesis/infrastructure/utils/FileUtils.java

```diff
@@ -152,4 +158,58 @@ public String getSpecFolder(String campaign) {
 	public String getDoneFolder(String campaign, String dataSource) {
 		return  String.format("%s/%s/%s/%s", dataFolderSource, "DONE", dataSource, campaign);
 	}
+
+	/**
+	 * Write a text file.
+	 * @param filePath Path to the file.
+	 * @param fileContent Content of the text file.
+	 */
+	public void writeFile(Path filePath, String fileContent) {
+		Path path = Path.of(dataFolderSource, filePath.toString());
+		boolean fileCreated = false;
+		File myFile = null;
```

Comment:
Should we close file ?

Reply:
The FileWriter is open in a try-with-resources so it closes after the execution of the block